hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -418,27 +657,81 @@ public:
    /// Normalized axes access, converting from actual axis type to base class
    const RAxisBase &GetAxis(int iAxis) const final { return *std::apply(Internal::GetAxisView<AXISCONFIG...>, fAxes)[iAxis]; }
 
-   /// Gets the bin index for coordinate `x`; returns -1 if there is no such bin,
+   /// 
+   std::vector<int> GetBinType(const CoordArray_t &x) const
+   {
+      return Internal::RGetOverflowBin<DATA::GetNDim() - 1, RHistImpl, decltype(fAxes)>()(nullptr, fAxes, x);
+   }
+
+   /// 
+   int GetOverflowOffset(int axis) const
+   {
+      return Internal::RGetOverflowOffset<DATA::GetNDim() - 1, decltype(fAxes)>()(fAxes, axis);
+   }
+
+   /// 
+   int GetAxisOffset(int axis) const
```

Comment:
That's an offset in the overflow bins, right? Then maybe `GetOverflowAxisOffset`?